### PR TITLE
Ensure project name input in cli shows validation errors

### PR DESCRIPTION
**Pull request by zephraph**
Added a little context for folks entering in invalid project names. Error message just links off to the docs to avoid having to do a lot of logic checking for the project name recommendation.

### CODE REVIEW

**Comment by transitive-bullshit** [comment]
Looks good. The only other option I'd consider would be throwing an error instead of returning a string.

Happy with either solution. Let's just make sure all the use cases are converted before merging.

**Comment by zephraph** [comment]
I'd be good w/ the error case. Will update.

File: packages/saasify-faas-utils/lib/validators.js [hunk 1]
```diff
@@ -30,7 +30,15 @@ exports.password = (value) => {
...
-  return value && exports.projectNameRe.test(value)
+  if (!value || value.length < 3) {
+    return 'Project name must be at least 3 characters'
+  } else if (value.length > 64) {
+    return 'Project name must be less than 64 characters'
+  } else if (!exports.projectNameRe.test(value)) {
+    return 'Project name can only be alphanumeric or a dash.'
+  } else {
+    return true
+  }
```
Comment (jamesgeorge007) [inline packages/saasify-faas-utils/lib/validators.js]:
```suggestion
  }
  return true
```

Comment (jamesgeorge007) [reply packages/saasify-faas-utils/lib/validators.js]: `else` is kinda redundant here.

**Pull request merged**